### ISSUE TITLE
App Crashing due to ANSMetaData

### ISSUE THREAD

**Issue by ioskunal** [SwiftKickMobile/SwiftMessages#204]
Does your code contain ANSMetaData Class that checks whether the device is Jail Broken or not?
Whenever I try to run the app on Jail Broken Device the application crashes on while displaying the success message.

**Comment by wtmoose** [comment]
No, it doesn't. This issue with jailbroken devices #9 was recently fixed. So make sure you're using the latest version. Otherwise, please supply a crash log and I'll take a look.
